docs/src/community/goversance.md

```diff
@@ -0,0 +1,34 @@
+# Code of Conduct
+
+Youki follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+# Maintainership
+
+Youki is governed by Maintainers who are elected from active contributors.
+Youki will keep the [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+Maintainers are [here](./maintainer.md).
+
+## Roles
+
+Maintainers are consist of the following roles:
```

Comment:
The user of the term maintainers and committers in the following section is a little confusing. If I understand correctly, maintainers == commiters. If so, I would keep the definition below using the word `commiter`, but standardize on the use of maintainers for the discussion of promotion, demotion, and etc.

Reply:
@yihuaf @Furisto 
Good point. I was planning to add another Role, Reviewers who are candidates for Comitter. But I didn't write it down then because it would take a little more time to sort out the relevant people.
But, well, I added it because I think it is good to write it down anyway.
https://github.com/containers/youki/pull/2806/commits/09e8cd173d096b919ac497f19fd5871965290442